### ISSUE TITLE
Can't open NBXplorer solution in VS2017 anymore

### ISSUE THREAD

**Issue by bazooka70** [dgarage/NBXplorer#58]
As I understand you moved to core 2.1 (?)
But now I no longer can open the solution in VS 2017 (15.5.7).
The VS suggests I should install some[ ".NET Core 2.1 Preview 1" ](https://github.com/aspnet/Tooling/blob/master/dotnet-core-2.1.md) boolshit. 
But I don't understand what that "Preview" means and I have many other projects to maintain so I'm a bit hasisated about that move...

Was it really necessary to move to 2.1? :/
Do you have any suggestions on this issue?

Thanks

**Comment by NicolasDorier** [comment]
yes I upgraded to .NET 2.1 rc recently to get all the goodness coming in .NET Core soon. (Specially Alpine images and proper ARM support)
